### DISCUSSION TITLE
Support For Usenet?

### DISCUSSION THREAD

**Discussion by DC651F82053AD314** [itsToggle/plex_debrid#518] [Ideas]
Hello, cool project!
Now that 'unlimited' storage is dead, the community needs something like this to turn to, but I think you can build more onto this project.

I know your project uses Torrents, but have you considered making Usenet an option too?  In my experience, Usenet is much faster and with no need to seed, an easier option than torrents.  Yes, I know you cannot stream files directly from Usenet, but here's my idea:

The user makes a selection on Plex, a countdown time estimation starts so the user knows approximately how much time it will take for the media content to be ready. After the media is downloaded/unpacked from Usenet, it is then 'staged' for viewing on the local server in the background.

After viewing, the downloaded media can then go into a local folder/pool on the server, where it sits incase it's called again.  The admin could allot a specified amount of storage to this temporary 'pool'.  Files in the pool would be overwritten as more media is pulled by users.

This would give the flexibility for people to use torrents and/or Usenet with your project. The downside, of course, is the time users will need to wait for the media to be downloaded, unpacked and ready for staging/viewing -- but the countdown would help users know about how long they'll need to wait.  Allowing the powerful use of Usenet, without needing a cloud storage provider.

If a project like this already exists, please let me know. I've been thinking about this idea for many years!

Cheers!

**Comment by gravelfreeman** [comment]
+1 for Usenet support.

I came here for this exact reason.

**Comment by profucius** [comment]
As much as I like Usenet and think it's a great resource, it would seem Usenet is outside the scope of this project. Personally I would rather the dev focus their efforts on improving and adding features to the already existing functionality. Spreading too thin across diverse features can cause devs to burn out and give up, and if it were to come to that, I'd rather they avoid it.

**Reply by gravelfreeman** [reply]
Why would it be outside of the scope of this project? I mean usenet makes much more sense than torrents imo.

**Comment by dreulavelle** [comment]
Definitely outside the scope of the app. Usenets need to get downloaded. They don't get added to real-debrid (or any debrid service that I'm aware of?). 

The whole point of this app is to curate an infinite library with unlimited storage since its all stored on Real-Debrid servers.
